NumWords.java

```diff
@@ -1,6 +1,14 @@
 // Prints a given number using a hundreds, tens, and units notation.
 public class NumWords {
 	public static void main(String args[]) {
-	    // Replace this comment with your code
+	    // clasify
+		int Num = Integer.parseInt ( args [0] );
+		int hundreds = Num / 100;
+		int tens = ( Num / 10 ) - ( hundreds * 10 );
```

Comment:
Can be simplified to:
int hundreds = num / 100;
int tens = (num % 100) / 10;
int ones = num % 10;